pkg/server/server.go

```diff
@@ -446,7 +446,7 @@ func (s *Server) Run(ctx context.Context) error {
 	}
 
 	if s.Options.Controllers.EnableAll || enabled.Has("quota") {
-		if err := s.installKubeQuotaController(ctx, controllerConfig, delegationChainHead); err != nil {
+		if err := s.installKubeQuotaController(controllerConfig, delegationChainHead); err != nil {
```

Comment:
I'm going to have to add this right back in this week for getting the logger from the context, do you mind not removing it for now?

Reply:
sure, no problem.

Reply:
done.